client/analytics/report/customers/config.js

```diff
@@ -167,6 +167,81 @@ export const advancedFilters = {
 				} ) ),
 			},
 		},
+		order_count: {
+			labels: {
+				add: 'No. of Orders',
+				remove: 'Remove order  filter',
+				rule: 'Select an order count filter match',
+				title: 'No. of Orders {{rule /}} {{filter /}}',
+			},
+			rules: [
+				{
+					value: 'max',
+					label: 'Less Than',
+				},
+				{
+					value: 'min',
+					label: 'More Than',
+				},
+				{
+					value: 'between',
+					label: 'Between',
+				},
+			],
+			input: {
+				component: 'Number',
+			},
+		},
+		total_spend: {
+			labels: {
+				add: 'Total Spend',
+				remove: 'Remove total spend filter',
+				rule: 'Select a total spend filter match',
+				title: 'Total Spend {{rule /}} {{filter /}}',
+			},
+			rules: [
+				{
+					value: 'max',
+					label: 'Less Than',
+				},
+				{
+					value: 'min',
+					label: 'More Than',
+				},
+				{
+					value: 'between',
+					label: 'Between',
+				},
+			],
+			input: {
+				component: 'Number',
```

Comment:
This will point to `<NumberFilter />`, which already has currency built in if the `type` prop is set to `currency`. Maybe we can support an input type, `component: 'Currency'`, [here](https://github.com/woocommerce/wc-admin/blob/2a729477b81758106d288a1e7352e3236e6ef72d/packages/components/src/filters/advanced/index.js#L190) which uses `<NumberFilter />` with a currency prop.

Reply:
Good idea.  Added this in https://github.com/woocommerce/wc-admin/pull/1116/commits/caed8340592e3152cc15184d5498a2a9e1d47faa